service/ztests/vector.yaml

```diff
@@ -0,0 +1,33 @@
+script: |
```

Comment:
We should add the local lake equivalent of this test in lake/ztests/vector.yaml so we have coverage for the lake/api/local.go change.

Reply:
This test is an exact copy of lake/ztests/vector.yaml

Reply:
My apologies. I totally missed that.